### ISSUE TITLE
Does lambda not supported?

### ISSUE THREAD

**Issue by Tatsuki-I** [i-tu/Hasklig#92]
Does lambda not supported?

**Comment by cblp** [comment]
How would you want lambda to be supported? Draw `\` as `λ`? But then how should `\λ` be rendered?

**Comment by BornForFever** [comment]
Disable `\` if it is followed by `λ`.

**Comment by Tatsuki-I** [comment]
I think that drawing `\` as `λ` is the best.

**Comment by BornForFever** [comment]
Yes. BTW, as for `\λ`, IMHO, will the `calt`(contextual alternative) feature help? I don't know much about OpenType features.
